tests/integration/testlib/ovslib.py

```diff
@@ -25,7 +25,6 @@
 from libnmstate.schema import InterfaceState
 from libnmstate.schema import InterfaceType
 from libnmstate.schema import OVSBridge
-from libnmstate.schema import OVSBridgePortType
 
 
 class Bridge:
```

Comment:
Ohhh... this is a bug :/

Reply:
I don't get it, what is the bug?

Reply:
You mean having Bridge without (object)?

Reply:
Yes. It will be fine when we drop py2. (Should happen soon)

Reply:
Fixed, don't look